modules/ROOT/pages/enabling-einstein.adoc

```diff
@@ -0,0 +1,113 @@
+
+As an organization administrator, you can assign permission to the users in your organization so they can use Einstein with Anypoint Platform products that have Einstein capabilities, including:
+
+* Anypoint Code Builder
+* Intelligent Document Processing (IDP)
+
+
+== Before You Begin
+
+Before enabling Einstein in your Anypoint Platform organization, you must:
+ 
+* Have administrative access to an Anypoint Platform organization that isn't a trial organization and isn't expired
+* Have the Organization Administrator permission and role
+* Have administrative access to a Salesforce organization that https://help.salesforce.com/s/articleView?id=sf.generative_ai_enable.htm&type=5[has Einstein enabled]. 
+* xref:access-management::trusted-salesforce-org.adoc[Establish a tenant relationship] between your Anypoint Platform organization and a trusted Salesforce organization. 
+
+[[enable-einstein-salesforce]]
+== Enable Einstein in the Salesforce Organization
+
+When the Salesforce administrator enables Einstein for the connected Anypoint Platform organization, Einstein requests are routed through the connected Salesforce organization. Billing and tracking are done in the Salesforce organization. 
+
+To enable Einstein for the connected Anypoint Platform organization:
+
+. Log in to your Salesforce organization.
+. In Salesforce Setup, go to the *Quick Find* box and enter `MuleSoft`, then select *MuleSoft > Anypoint Platform Setup*.
+. In *Your connected Anypoint Platform Organization*, click *View Connection Info* to open a new dialog.
+. In *Connection to MuleSoft Anypoint Platform*, switch between *Enabled* or *Disabled* in *Enable Einstein for MuleSoft Anypoint Platform*.
++ 
+image::einstein-enabled-in-salesforce.png["Einstein for MuleSoft Anypoint Platform is enabled"]
+. Click *Close*.
++
+After Einstein is enabled in the Salesforce organization, it's displayed in Anypoint Platform as *Enabled in Salesforce* for the connected Salesforce organization.
+
+[[enable-einstein-anypoint]]
+== Enable Einstein for Your Anypoint Platform Organization
+
+When Einstein is enabled in Anypoint Platform, any Anypoint Platform feature running in that organization is able to use Einstein. The Salesforce administrator can't specify which features are enabled for Einstein, for example, IDP can use Einstein but Anypoint Code Builder can't. 
+
+To enable Einstein for your Anypoint Platform organization:
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. Click *Salesforce*.
+. In *Enable Einstein for Anypoint Platform*, click *Accept*.
+. Read the terms and conditions, select *I accept the terms and conditions.", and then click *Accept*.
++
+Salesforce orgs that have xref:connecting-salesforce-orgs.adoc[established tenant relationships] with your Anypoint Platform organization are listed in the *Salesforce Orgs* section. 
```

Comment:
```suggestion
The *Salesforce Orgs* section shows Salesforce orgs that have xref:connecting-salesforce-orgs.adoc[established tenant relationships] with your Anypoint Platform organization. 
```

Reply:
Consider using "org" instead of "organization" in some other earlier mentions.

Reply:
I started using org, but I think I changed them all to organization (obvs missed this one). We have never used "org" for Access Management. Salesforce org is common in Salesforce documentation. But it seems strange to mix org and organization. What do you think @hannanelson ?